src/commands/parser.rs

```diff
@@ -507,6 +507,17 @@ pub struct Migrate {
     /// Do not print any messages, only indicate success by exit status
     #[clap(long)]
     pub quiet: bool,
+
+    /// Upgrade to a specified revision.
+    ///
+    /// Unique prefix of the revision can be specified instead of full
+    /// revision name.
+    ///
+    /// If this revision is applied, the command is no-op. The command
+    /// ensures that this revision present, but it's not an error if more
+    /// revisions are applied on top.
```

Comment:
```suggestion
    /// will also apply all unapplied revisions that precede the specified
    /// revision.
```

Reply:
Well, this is not what I meant. I mean that if you have a DB with applied revision 4 and you specified `--no-revision=2` this is no-op. It doesn't fail saying that newer revision is present in the database.